IdokladSdk/Models/IssuedInvoice/Post/IssuedInvoicePostModel.cs

```diff
@@ -65,7 +65,7 @@ public class IssuedInvoicePostModel : ValidatableModel
         /// <summary>
         /// Gets or sets date of VAT application.
         /// </summary>
-        [DateGreaterOrEqualThan(Constants.DefaultDateTimeString)]
+        [DateGreaterOrEqualThan(Constants.DefaultDateTimeString, true)]
```

Comment:
Nechápu, proč je tam allowNull = true když ta oroperta není nullable. V API taky není nullable, proč je tam tedy to allowNull?